tests/test_on_data/ecephys/test_spikeglx_converter.py

```diff
@@ -193,7 +194,7 @@ def test_electrode_table_writing(tmp_path):
     np.testing.assert_array_equal(saved_channel_names, expected_channel_names_nidq)
 
     # Test AP
-    electrical_series = nwbfile.acquisition["ElectricalSeriesAPImec0"]
+    electrical_series = nwbfile.acquisition["ElectricalSeriesAP"]
```

Comment:
Previously, the Converter always writes the ElectricalSeries using the name `f"ElectricalSeriesAPImec{probe_index}"` whereas the interface uses the name `ElectricalSeriesAP` only. This PR makes the behavior of the converter and the interface consistent in the single probe case hence the change.